src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -499,16 +499,16 @@ def test_remove_osds(self, cephadm_module):
             out = wait(cephadm_module, c)
             assert out == ["Removed osd.0 from host 'test'"]
 
-            cephadm_module.to_remove_osds.enqueue(OSD(osd_id=0,
-                                                      replace=False,
-                                                      force=False,
-                                                      hostname='test',
-                                                      fullname='osd.0',
-                                                      process_started_at=datetime.datetime.utcnow(),
-                                                      remove_util=cephadm_module.rm_util
-                                                      ))
+            cephadm_module.to_remove_osds.put(OSD(osd_id=0,
+                                                  replace=False,
+                                                  force=False,
+                                                  hostname='test',
+                                                  fullname='osd.0',
+                                                  process_started_at=datetime.datetime.utcnow(),
+                                                  remove_util=cephadm_module.rm_util
+                                                  ))
             cephadm_module.rm_util.process_removal_queue()
-            assert cephadm_module.to_remove_osds == OSDQueue()
+            assert isinstance(cephadm_module.to_remove_osds, OSDQueue)
```

Comment:
I think this was for checking that the queue was empty, rather than the class...

Reply:
idd